### ISSUE TITLE
Add a setting to exclude an extension from auto-update

### ISSUE THREAD

**Issue by serbani** [microsoft/vscode#144395]
It would be useful to have a setting to exclude some extensions from auto-update. There are a couple of scenarios in which this would be desirable:
- an extension is critical for your workflow and you want to avoid any undesired regressions or changes that might affect workflow at a bad time
- keep ion sync all the version of an extension used in a team, it might not update for everybody at the same time
- an extension becomes incompatible with your OS after an update

**Comment by tolisaverkios** [comment]
Yes I agree. Currently, the "Extensions: Auto update" user setting has only options for "None", "All extensions", "Only enabled Extensions".

It would be useful if the users can decide which extensions they want to have manual update, and keep the rest of the extensions with auto-update

**Comment by sandy081** [comment]
/duplicate

https://github.com/microsoft/vscode/issues/131733
